### ISSUE TITLE
Privacy Policy link fails WCAG contrast requirements

### ISSUE THREAD

**Issue by HamishSlater** [StarCitizenWiki/mediawiki-extensions-EmbedVideo#133]
**Describe the bug**
The Privacy Policy link has insufficient text contrast against the background. This fails WCAG 2.1 AA requirements.
See: https://www.w3.org/TR/WCAG21/#contrast-minimum

**To Reproduce**
1. Click 'Load video'
2. Observe contrast ratio of 1.15:1

**Expected behavior**
Contrast ratio of at least 4.5:1

**Technical information:**
 - Browser: Chrome/Firefox
 - Extension Version: 3.2.6 (master)
 - MediaWiki Version: 1.43.4
